packages/scheduler/src/components/Conflict/Conflict.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {intlShape} from 'react-intl'
+import {conflicts} from './utils'
+
+const Conflict = ({conflictStatus, intl}) => {
+  if (!(conflictStatus === conflicts.ACCEPTED || conflictStatus === conflicts.EXISTING)) {
+    return null
+  }
+
+  const accepted = conflictStatus === conflicts.ACCEPTED
+
+  const icon = accepted ? 'fa-check' : 'fa-times'
+  const textResource = `client.scheduler.conflict${accepted ? 'Accepted' : 'Existing'}`
+  const style = {
+    ...(accepted ? {} : {color: '#8b0000'})
+  }
+
+  return <span style={style}>
+    <i className={`fa ${icon}`} style={{paddingRight: '10px'}} aria-hidden="true"></i>
+    {intl.formatMessage({id: textResource})}
+  </span>
+}
+
+Conflict.propTypes = {
+  intl: intlShape.isRequired,
+  conflictStatus: PropTypes.string
```

Comment:
Maybe better to restrict the `conflictStatus` prop to one of the status defined in utils.js so that a warning/error appears in the console if an unexpected prop is defined? Otherwise it could be overlooked here easily if a status changes or a new one is added.